tests/test_parser/test_properties/test_model_property.py

```diff
@@ -29,6 +29,9 @@ def test_get_type_string(no_optional, nullable, required, expected):
         required_properties=[],
         relative_imports=set(),
         additional_properties=False,
+        read_only=False,
+        discriminator_property=None,
+        discriminator_mappings={},
```

Comment:
Those last two lines aren't relevant here, I just had to add them due to the quirks of how `attrs` handles properties with default values when subclasses are involved.